handler/src/main/java/io/netty/handler/ipfilter/IpFilterRuleType.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2014 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.handler.ipfilter;
+
+/**
+ * Used in {@link IpFilterRule} to decide if a matching IP Address should be allowed or denied to connect.
+ */
+public enum IpFilterRuleType {
+    Accept,
+    Reject
```

Comment:
Shouldn't we use `ACCEPT` and `REJECT`? If we have the same problem in 4.1 and master, we should fix them, too.